libs/core/src/lib/token/tokenizer.component.ts

```diff
@@ -115,12 +115,25 @@ export class TokenizerComponent implements AfterViewChecked, AfterViewInit, Afte
     /** @hidden */
     hiddenCozyTokenCount: number = 0;
 
+    /** @hidden */
+    shiftKeyPressed: boolean = false;
+
+    /** @hidden */
+    ctrlKeyPressed: boolean = false;
+
+
     /** @hidden */
     ngAfterViewInit(): void {
         if (this.input && this.input.elementRef()) {
             this.input.elementRef().nativeElement.addEventListener('keydown', (event) => {
                 this.handleKeyDown(event, this.tokenList.length);
             });
+            this.tokenizerInnerEl.nativeElement.addEventListener('keydown', (event) => {
+              this.handleKeyDown(event, this.tokenList.length);
+          });
+            this.tokenizerInnerEl.nativeElement.addEventListener('keyup', (event) => {
+              this.handleKeyUp(event);
+          });
```

Comment:
Let's move it to some private method `_setUpKeyboardHandler()` or something similar